code/upaya/Gemfile

```diff
@@ -55,7 +55,7 @@ group :development do
   gem 'aws-sdk', '~> 2.0'
   gem 'better_errors'
   gem 'derailed'
-  gem 'binding_of_caller', platforms: [:mri_21]
+  gem 'binding_of_caller', platforms: [:mri_23]
```

Comment:
`mri_23` is not a valid platform. I found out when running `bundle install` 😄 Officially the gem doesn't support Ruby greater than 2.1, but I saw a pull request that shows tests are passing in 2.2. Don't know about 2.3. I suggest we keep it, but remove the platforms options so that we don't restrict the version where it runs.